www/docs/usage/commandline.md

```diff
@@ -17,47 +17,74 @@ The most recently selected versions are presented at the top of the dropdown.
 ## See all versions including beta, alpha and release candidates(rc)
 <img src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/tfswitch-v5.gif" alt="drawing" style="width: 600px;"/>
 
-1. Display all versions including beta, alpha and release candidates(rc). 
+1. Display all versions including beta, alpha and release candidates(rc).
 2. For example, `tfswitch -l` or `tfswitch --list-all` to see all versions.
 3. Hit **Enter** to select the desired version.
 
 ## Use environment variables
-You can also set the `TF_VERSION` environment variable to your desired terraform version. 
-For example:   
+
+You can also set the `TF_VERSION` environment variable to your desired terraform version.
+For example:
+
 ```bash
 export TF_VERSION=0.14.4
 tfswitch #will automatically switch to terraform version 0.14.4
 ```
+
 ## Install latest version only
+
 1. Install the latest stable version only.
 2. Run `tfswitch -u` or `tfswitch --latest`.
 3. Hit **Enter** to install.
 
 ## Install latest implicit version for stable releases
+
 1. Install the latest implicit stable version.
 2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads 0.13.6 (latest) version.
 3. Hit **Enter** to install.
 
 ## Install latest implicit version for beta, alpha and release candidates(rc)
+
 1. Install the latest implicit pre-release version.
 2. Ex: `tfswitch -p 0.13` or `tfswitch --latest-pre 0.13` downloads 0.13.0-rc1 (latest) version.
 3. Hit **Enter** to install.
 
 ## Show latest version only
+
 1. Just show what the latest version is.
 2. Run `tfswitch -U` or `tfswitch --show-latest`
 3. Hit **Enter** to show.
 
 ## Show latest implicit version for stable releases
+
 1. Show the latest implicit stable version.
 2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` shows 0.13.6 (latest) version.
 3. Hit **Enter** to show.
 
 ## Show latest implicit version for beta, alpha and release candidates(rc)
+
 1. Show the latest implicit pre-release version.
 2. Ex: `tfswitch -P 0.13` or `tfswitch --show-latest-pre 0.13` shows 0.13.0-rc1 (latest) version.
 3. Hit **Enter** to show.
 
-## Use custom mirror 
-To install from a remote mirror other than the default(https://releases.hashicorp.com/terraform). Use the `-m` or `--mirror` parameter.    
-Ex: `tfswitch --mirror https://example.jfrog.io/artifactory/hashicorp`
+## Use custom mirror
+
+To install from a remote mirror other than the default(https://releases.hashicorp.com/terraform). Use the `-m` or `--mirror` parameter.
+
+```bash
+tfswitch --mirror https://example.jfrog.io/artifactory/hashicorp`
+```
+
+## Install to non-default location
+
+By default `tfswitch` will download the Terraform binary to the user home directory under this path: `/Users/warrenveerasingam/.terraform.versions`
```

Comment:
It's sort of misleading to use Warren's username and if user is on Linux or Windows. What do you think of this commit suggestion?
```suggestion
By default `tfswitch` will download the Terraform binary to the user home directory under this path: `$HOME/.terraform.versions`
```

Reply:
Fixing